src/api.rs

```diff
@@ -2476,39 +2432,6 @@ pub struct Monitor {
     pub status: String,
 }
 
-#[derive(Debug, Serialize, Deserialize)]
-#[serde(rename_all = "snake_case")]
-pub enum MonitorCheckinStatus {
```

Comment:
This stuff is now available in `sentry-types` after https://github.com/getsentry/sentry-rust/pull/577